### PR TITLE
feat: Add VirtualBox kmod

### PR DESCRIPTION
**Pull request by EyeCantCU**
Adds the kernel module needed for VirtualBox

Resolves: https://github.com/ublue-os/akmods/issues/105

### CODE REVIEW

**Review by HikariKnight — APPROVED** [review]
looks good

**Comment by fiftydinar** [comment]
Add this module to README

File: build-kmod-VirtualBox.sh [hunk 1]
```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+
+set -oeux pipefail
+
+cp /tmp/ublue-os-akmods-addons/rpmbuild/SOURCES/_copr_ublue-os-akmods.repo /etc/yum.repos.d/
```
Comment (bsherman) [inline build-kmod-VirtualBox.sh:5]: Is this copr repo needed since virtualbox akmod is in rpmfusion?

Comment (EyeCantCU) [reply build-kmod-VirtualBox.sh:5]: You are correct. Will remove it

**Review by bsherman — APPROVED** [review]
Looks good.

**Review by HikariKnight — APPROVED** [review]

**Pull request merged**